purchase_order_report_grouped_by_vendor/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order Report Grouped By Vendor",
+    "summary": "Purchase order report grouping orders by vendor",
+    "version": "15.0.1.0.0",
+    "category": "Purchase",
+    "website": "https://github.com/OCA/purchase-reporting",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "purchase_requisition",
```

Comment:
We should avoid this dependency for the generic module. Add this reference in custom. Put a hook method here if needed.

Reply:
The dependency was wrong, I should have changed it before pushing the module to the repository. 
All changes done, thanks